telemetry/definitions/commonDefinitions.json

```diff
@@ -1385,6 +1385,11 @@
             "type": "string",
             "description": "User locale. Examples: en-US, en-GB, etc."
         },
+        {
```

Comment:
can we reuse id instead of declaring "metricId"

Reply:
that seems to vague when we have multiple types of IDs